src/events/components/DetailView/AttendanceEvent.tsx

```diff
@@ -70,6 +72,33 @@ const AttendanceEvent: FC<IProps> = ({ eventId, eventTitle }) => {
       <Block title="Venteliste">
         <p>{attendanceEvent.waitlist ? attendanceEvent.number_on_waitlist : '-'}</p>
       </Block>
+      {attendanceEvent.has_extras && attendanceEvent.is_attendee && (
+        <Block title="Ekstras">
+          <Select>
+            <option
+              key={-1}
+              value={-1}
+              onClick={(e: React.MouseEvent<HTMLOptionElement, MouseEvent>) =>
+                dispatch(patchAttendee({ attendeeId: attendee!.id, extras: Number(e.currentTarget.value) }))
+              }
+            >
+              Velg ekstra
+            </option>
+            {attendanceEvent.extras.map((extra: IExtra) => (
+              <option
+                key={extra.id}
+                value={extra.id}
+                selected={extra.id == attendee!.extras}
```

Comment:
Seems like ESLint didn't like this syntax. That's kinda weird, since the local check did not complain..! We should probably ensure that they are using the same config.
Either way, decide if you wanna fix the warning or just stick with this syntax!